src/WorkerChannel.ts

```diff
@@ -208,14 +211,15 @@ export class WorkerChannel implements IWorkerChannel {
               }
             }
           }
-          // Data from return supersedes data from context.bindings
-          if (result.bindings && !response.outputData) {
-            response.outputData = Object.keys(info.outputBindings)
-              .filter(key => result.bindings[key] !== undefined)
+          // Set results from context.bindings
+          if (result.bindings) {
+            response.outputData = response.outputData.concat(Object.keys(info.outputBindings)

```

Comment:
>response.outputData.concat [](start = 34, length = 26)

From looking up, I think concat flattens things, right? 

for instance,
context.bindings.outputEventMessage = [1,2]
return {
   res: {body: [3,4]}
}

do you want [1,2,3,4] to be returned or does it make sense to return it as [[1,2],[3,4]]? I don't know if this example clearly indicates what I'm trying to say / if this is even a valid example...

Reply:
@yojagad - One thing to note is that every element in outputData is mapped to an object with name and data properties, so:
outputEventMessage maps to:
```
{
  name: outputEventMessage,
  data: [1, 2]
}
```
and res maps to (roughly... data is actually converted to a different type):
```
{
  name: res,
  data: { body: [3, 4] }
}
```
So the concat is between 
```
[{
  name: outputEventMessage,
  data: [1, 2]
}]
```
and
```
[{
  name: res,
  data: { body: [3, 4] }
}]
```
which I want to become
```
[{
  name: outputEventMessage,
  data: [1, 2]
},
{
  name: res,
  data: { body: [3, 4] }
}]
```